src/utils.test.ts

```diff
@@ -1,49 +1,47 @@
 import test from "ava"
-import { safeParseJSON } from "./utils"
-import { Buffer } from "buffer/"
+import { safeUserParse } from "./utils"
 
 let userNames = [
-  `【パンツ】Pantsu`,
-  `🎃 ASH  П (◣_◢) П  🎃`,
-  `七曜曜`,
-  `Dunesca ʕ•͡ᴥ•ʔ`,
+  `【パンツ】Pantsu#1234`,
+  `🎃 ASH  П (◣_◢) П  🎃#1234`,
+  `七曜曜#1234`,
+  `Dunesca ʕ•͡ᴥ•ʔ#1234`,
+  `Deadsiesthe•̪̀●́#1234`,
+  `Deadsiesthe"*�#1234`
 ]
 
-function toB64(str: string): string {
-  return Buffer.from(str, "ascii").toString("base64")
-}
-
-function fromB64(str: string): string {
-  return Buffer.from(str, "base64").toString()
-}
-
-function encode(obj: Record<string, unknown>): string {
-  return fromB64(toB64(JSON.stringify(obj)))
-}
```

Comment:
This part was to mimic the mangling fatshark are doing. e.g.

```js
JSON.parse(`{"AccountName":"【パンツ】Pantsu#1234"}`)
```
is fine but
```js
JSON.parse(fromB64(toB64(`{"AccountName":"【パンツ】Pantsu#1234"}`)))
```
errors

Reply:
Ah, you're right - I didn't realise you'd already basically worked out how they're mangling it, would've saved me some time!

Will re-add and give it a comment for future users